bitcoin/script/miniscript.cpp

```diff
@@ -330,6 +330,9 @@ bool DecomposeScript(const CScript& script, std::vector<std::pair<opcodetype, st
             opcode = OP_VERIFY;
         } else if (IsPushdataOp(opcode)) {
             if (!CheckMinimalPush(push_data, opcode)) return false;
+        } else if (it != itend && (opcode == OP_CHECKSIG || opcode == OP_CHECKMULTISIG || opcode == OP_EQUAL) && (static_cast<opcodetype>(*it) == OP_VERIFY)) {
```

Comment:
I don't think this `static_cast` is needed.

Reply:
Right, removed it.